test/integration/middleware-src/test/index.test.js

```diff
@@ -18,23 +18,19 @@ const appDir = join(__dirname, '../')
 const srcHeader = 'X-From-Src-Middleware'
```

Comment:
Can revert this flakiness fix if we land https://github.com/vercel/next.js/pull/53769 instead